docs/accessing-elastic-services.asciidoc

```diff
@@ -229,7 +229,22 @@ Now you should get this message:
 curl: (51) SSL: no alternative certificate subject name matches target host name '35.198.131.115'
 ----
 
-Add the external IP of the service to the SANs of the certificate in the same Elasticsearch resource YAML manifest used to create the cluster and apply it again using `kubectl`.
+As the certificate subject by default is service name we can resolve this by using the service name 
+and specify the resolve information through an additional argument.
+
+[source,sh]
+----
+> curl --cacert tls.crt -u elastic:$PW https://$NAME-es-http:9200/ --resolve $NAME-es-http:9200:$IP
+{
+  "name" : "hulk-es-4qk62zd928",
+  "cluster_name" : "hulk",
+  "cluster_uuid" : "q6itjqFqRqW576FXF0uohg",
+  "version" : {...},
+  "tagline" : "You Know, for Search"
+}
+----
+
+Alternatively, and for other clients where the resolve information can't be specified, or when it is more convenient modify the server side, you can add the external IP of the service to the SANs of the certificate in the same Elasticsearch resource YAML manifest used to create the cluster and apply it again using `kubectl`.
```

Comment:
I think there may be a simpler way to phrase this. Maybe
"As an alternative, you can edit the Elasticsearch spec to include the external IP of the service to the certificate subject alternative names (SANs) as described below. This is useful for other clients where resolve information cannot be specified, or when it is more convenient to make changes on the server rather than client." ?